fetch.bs

```diff
@@ -7800,11 +7819,16 @@ method steps are:
  <li><p>Return <var>p</var>.
 </ol>
 
-<p>To <dfn>abort fetch</dfn> with a <var>promise</var>, <var>request</var>, and
-<var>responseObject</var>, run these steps:
+<p>To <dfn>abort fetch</dfn> with a <var>promise</var>, <var>request</var>,
+<var>responseObject</var>, and an <var>error</var>, run these steps:
 
 <ol>
- <li><p>Let <var>error</var> be an "<code><a exception>AbortError</a></code>" {{DOMException}}.
+ <li><p>Let <var>serializedError</var> be the result of calling [$StructuredSerialize$]
+ with <var>error</var>. If that threw an exception, let <var>serializedError</var> be the result of
+ calling [$StructuredSerialize$] with "{{AbortError}}" {{DOMException}}.
+
+ <li><p>Set <var>controller</var>'s <a for="fetch controller">serialized abort reason</a> to
```

Comment:
_controller_ is not declared.

Overall, I don't *think* these steps are necessary:

- If this is called from fetch() 4.1, then we are doing an early-abort; nothing will ever reach the server worker, and no controller will be involved.

- If this is called from fetch 11.2, then step 11.3 will take care of the serialization and controller-storage.

- If this is called from fetch 12.2, then... well, probably this case we do need to modify the controller, but maybe we should do it like step 11.2 + 11.3 and keep them separate? I.e. if response's aborted flag is set, then: 1. create a new "AbortError" DOMException; 2. "abort fetch"; 3. "abort controller"?

Reply:
That makes sense to me! I've added some substeps to fetch 12.2 to address that.